server/views/admin-add-article.hbs

```diff
@@ -14,11 +14,43 @@
             {{/each}}
         </select>
     </div>
+        <label for="example-text-input text-success " class="col-12 col-form-label">Language:</label>
+
+    <div class="form-check form-check-inline">
+
+        <label class="form-check-label">
+            <input class="form-check-input" type="checkbox" id="inlineCheckbox1" name="language" value="En" checked> English
+        </label>
+    </div>
+    <div class="form-check form-check-inline">
+        <label class="form-check-label">
+            <input class="form-check-input" type="checkbox" id="inlineCheckbox2" name="language" value="Tig"> Tigrenya
+        </label>
+    </div>
+    <div class="form-check form-check-inline">
+        <label class="form-check-label">
+            <input class="form-check-input" type="checkbox" id="inlineCheckbox3" name="language" value="Ar"> Arabic
+        </label>
+    </div>
+
+
     <div class="form-group has-success">
         <label class="text-success">Full Content:</label>
         <textarea class="form-control form-control-success mb-4" id="inputSuccess1" name="fullContent"></textarea>
-        <input class="btn btn-success" type="submit" value="Submit">
     </div>
 
+    <div class="form-group has-success">
```

Comment:
you wouldn't  need the input for three languages in the same form with this setup. Just have a dropdown list where you choose the language and one field for the content (that will be for that specific language)

Reply:
Alright I added this to test if the article list will show the article if i change the language